src/components/input-number/input-number.vue

```diff
@@ -247,7 +248,10 @@
             },
             setValue (val) {
                 // 如果 step 是小数，且没有设置 precision，是有问题的
-                if (!isNaN(this.precision)) val = Number(Number(val).toFixed(this.precision));
+                if(val){
```

Comment:
If we do `instance.setValue(0);` this will be falsy.

Reply:
If set value of 0, I want display 0 not 0.00.